core/src/test/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModelSpec.scala

```diff
@@ -0,0 +1,59 @@
+package org.broadinstitute.dsde.rawls.model
+
+import bio.terra.workspace.model.{WsmPolicyInput, WsmPolicyPair}
+import org.scalatest.flatspec.AnyFlatSpecLike
+import org.scalatest.matchers.should.Matchers
+import org.scalatestplus.mockito.MockitoSugar
+
+import java.util
+
+class WorkspaceModelSpec extends AnyFlatSpecLike with MockitoSugar with Matchers {
```

Comment:
Is there a reason this needs to be in a separate file vs [here](https://github.com/broadinstitute/rawls/blob/develop/model/src/test/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModelSpec.scala#L11)?

Reply:
it uses artifacts that are only available in the core project (specifically the WSM objects)